05-26 14:15:56 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Неизвестный тип данных: ")"
Unknown data type: ")"; SQL statement:
CREATE TABLE CITY(ID INT, NAME VARCHAR,REGION VARCHAR,DISTRICT VARCHAR ,POPULATION INT,FOUNDATION ) [50004-161]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.command.Parser.parseColumnWithType(Parser.java:3645)
	at org.h2.command.Parser.parseColumnForTable(Parser.java:3508)
	at org.h2.command.Parser.parseCreateTable(Parser.java:5174)
	at org.h2.command.Parser.parseCreate(Parser.java:3802)
	at org.h2.command.Parser.parsePrepared(Parser.java:324)
	at org.h2.command.Parser.parse(Parser.java:279)
	at org.h2.command.Parser.parse(Parser.java:251)
	at org.h2.command.Parser.prepareCommand(Parser.java:217)
	at org.h2.engine.Session.prepareLocal(Session.java:415)
	at org.h2.engine.Session.prepareCommand(Session.java:364)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1119)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:164)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:152)
	at dao.DataSource.connection(DataSource.java:25)
	at controller.UserMenu.menu(UserMenu.java:52)
	at Main.main(Main.java:5)
05-26 14:18:38 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Таблица "CITY" уже существует
Table "CITY" already exists; SQL statement:
CREATE TABLE CITY(ID INT, NAME VARCHAR,REGION VARCHAR,DISTRICT VARCHAR ,POPULATION INT,FOUNDATION INT) [42101-161]
05-26 14:19:00 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Таблица "CITY" уже существует
Table "CITY" already exists; SQL statement:
CREATE TABLE CITY(ID INT, NAME VARCHAR,REGION VARCHAR,DISTRICT VARCHAR ,POPULATION INT,FOUNDATION INT) [42101-161]
05-26 14:19:21 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Таблица "CITY" уже существует
Table "CITY" already exists; SQL statement:
CREATE TABLE CITY(ID INT, NAME VARCHAR,REGION VARCHAR,DISTRICT VARCHAR ,POPULATION INT,FOUNDATION INT) [42101-161]
05-26 14:19:48 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Таблица "CITY" уже существует
Table "CITY" already exists; SQL statement:
CREATE TABLE CITY(ID INT, NAME VARCHAR,REGION VARCHAR,DISTRICT VARCHAR ,POPULATION INT,FOUNDATION INT) [42101-161]
05-26 14:21:23 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "NSERT INTO TEST VALUES(1,'Адыгейск','Адыгея',[*]'Южный,12248,1973)"
Syntax error in SQL statement "NSERT INTO TEST VALUES(1,'Адыгейск','Адыгея',[*]'Южный,12248,1973)"; SQL statement:
NSERT INTO TEST VALUES(1,'Адыгейск','Адыгея','Южный,12248,1973) [42000-161]
05-26 14:21:57 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "NSERT[*] INTO TEST VALUES(1,'Адыгейск','Адыгея','Южный',12248,1973)"
Syntax error in SQL statement "NSERT[*] INTO TEST VALUES(1,'Адыгейск','Адыгея','Южный',12248,1973)"; SQL statement:
NSERT INTO TEST VALUES(1,'Адыгейск','Адыгея','Южный',12248,1973) [42000-161]
05-26 14:22:31 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "NSERT[*] INTO CITY VALUES(1,'Адыгейск','Адыгея','Южный',12248,1973)"
Syntax error in SQL statement "NSERT[*] INTO CITY VALUES(1,'Адыгейск','Адыгея','Южный',12248,1973)"; SQL statement:
NSERT INTO CITY VALUES(1,'Адыгейск','Адыгея','Южный',12248,1973) [42000-161]
